lib/providers/ModuleApiProvider.js

```diff
@@ -24,7 +24,7 @@ class ModuleApiProvider extends ModuleApiProviderBase {
 		 */
 		this.actions = {
 			redirectedTo: '',
-			isNotFoundCalled: false,
+			errorStatusCode: 200,
```

Comment:
I meant to name it `statusCode` and use it always writing the headers. Also, implement the redirect using this status code as well.